packages/admin/src/views/Admin.js

```diff
@@ -8,34 +8,10 @@
 
 import { useNavigate, Link } from "react-router-dom"
 import { useServices, Navbar, FooterSmall } from "@kernel/common"
+import AppConfig from "../App.config"
```

Comment:
We're using absolute imports [0] in the web apps, so should just be

```
import AppConfig from 'App.config'
```

[0] https://create-react-app.dev/docs/importing-a-component/#absolute-imports